Makefile

```diff
@@ -1,3 +1,8 @@
+BASE_PATH = $(shell pwd)
+BIN_PATH := $(BASE_PATH)/bin
+
+export PATH := $(BIN_PATH):$(PATH)
+
```

Comment:
Why do we need this?

Reply:
are you about all the variables?

Reply:
Yep, we have several new variables, but nothing changed in fact.

Reply:
Discussed and resolved